OneCommunityGlobal/HGNRest
# Jatin toggle invisibility permission self and others

# Description
![Screenshot 2024-11-05 at 10 10 49 AM (2)](https://github.com/user-attachments/assets/a1b7f4a3-40b8-4d24-8e75-605b4df0d945)

## Related PRS (if any):
This PR is related to [PR#2183](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2183) + [PR#906](https://github.com/OneCommunityGlobal/HGNRest/pull/906/)
This backend PR is related to the[ #2841](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2841) backend PR.

## Main changes explained:
- Fixed errors in src/components/userprofile.jsx

## How to test:
1. check into current branch
2. do `npm install` and `npm run build` and `npm start`to run this PR locally
3. Follow instructions on frontend.

## Screenshots or videos of changes:
https://github.com/user-attachments/assets/fed1be0c-d2d3-4d0d-9f00-ee9bfb4bf914

## Note:
Include the information the reviewers need to know.
